### PR TITLE
Jumphost

### PR DESCRIPTION
**Pull request by gmauro**
Added a jumphost configuration.
Now the central manager has two network interfaces, one on the public network and the other on a private network (192.168.199.0/24)

### CODE REVIEW

File: nfs.tf [hunk 1]
```diff
@@ -1,10 +1,15 @@
 resource "openstack_compute_instance_v2" "nfs-server" {
+  depends_on = ["openstack_networking_subnet_v2.internal"]
```
Comment (hexylena) [inline nfs.tf]: is this necessary, with the `network` block below? my understanding was it would wait properly?

**Review by hexylena — APPROVED** [review]
Looks like some good improvements, amazed the networking functions so easily!

File: nfs.tf (hunk 1 above)
Comment (gmauro) [reply nfs.tf]: Probably not. I can easily run a test without it today.

Comment (hexylena) [reply nfs.tf]: Anyway, looks good from my end, glad this worked so easily!

Comment (gmauro) [reply nfs.tf]: Not needed.

**Pull request merged**